crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -116,7 +116,10 @@ impl SimpleFaucet {
             .map(|q| GasCoin::try_from(&q.1).unwrap())
             .filter(|coin| coin.0.balance.value() >= (config.amount * config.num_coins as u64))
             .collect::<Vec<GasCoin>>();
+        let balance = coins.iter().map(|coin| coin.0.balance.value()).sum::<u64>();
         let metrics = FaucetMetrics::new(prometheus_registry);
+        // set initial balance when faucet starts
+        metrics.balance.set(balance as i64);
```

Comment:
Out of interest, is it possible to sum it as an `i64` to begin with?

Reply:
👍

Reply:
That does not work unfortunately as `balance.value()` in the lines above is `u64`, so I can only sum it to `u64`.